utils/dynamicLink.js

```diff
@@ -13,9 +13,24 @@ const generateLongDynamicLink = async (redirectDesktopLink, redirectAndroidLink
     if (redirectAndroidLink === null) redirectAndroidLink = redirectDesktopLink
     if (redirectIosLink === null) redirectIosLink = redirectDesktopLink
 
-    const longDynamicLink = `${FIREBASE_DYNAMIC_LINK_URL}?&apn=${FIREBASE_DYNAMIC_LINK_ANDROID_APP_PACKAGE}&afl=${redirectAndroidLink}&isi=${BETTER_APP_STORE_ID}&ibi=${FIREBASE_DYNAMIC_LINK_IOS_APP_PACKAGE}&ifl=${redirectIosLink}&efr=1&ofl=${redirectDesktopLink}`
+    const longDynamicLink = `${FIREBASE_DYNAMIC_LINK_URL}?&apn=${FIREBASE_DYNAMIC_LINK_ANDROID_APP_PACKAGE}&afl=${redirectAndroidLink}&isi=${BETTER_APP_STORE_ID}&ibi=${FIREBASE_DYNAMIC_LINK_IOS_APP_PACKAGE}&ifl=${redirectIosLink}&efr=1&ofl=${redirectDesktopLink}&d=1`
+
     const { shortLink } = await firebaseDynamicLinks.createLink({
-        longDynamicLink
+        dynamicLinkInfo: {
+            domainUriPrefix: `${FIREBASE_DYNAMIC_LINK_URL}`,
+            link: longDynamicLink,
+            androidInfo: {
+                androidPackageName: FIREBASE_DYNAMIC_LINK_ANDROID_APP_PACKAGE,
+                androidFallbackLink: redirectAndroidLink
+            },
+            iosInfo: {
+                iosBundleId: FIREBASE_DYNAMIC_LINK_IOS_APP_PACKAGE,
+                iosFallbackLink: redirectIosLink
+            }
+        },
+        suffix: {
+            option: 'SHORT'
+        }
```

Comment:
Please add navigationInfo and enableForcedRedirect: true

```javascript
dynamicLinkInfo: {
       ...
        iosInfo: {
          iosBundleId: firebaseDynamicLinkIOSAppPackage
        },
        navigationInfo: {
          enableForcedRedirect: true
        }
      }
```

Reply:
updated mas